hello.html

```diff
@@ -1 +1,8 @@
+<HTML>
+<head>
```

Comment:
Thanks for using lowercase for these internal tags. Should HTML still be uppercase?